recipes/libtiff/all/conanfile.py

```diff
@@ -33,15 +32,25 @@ def source(self):
         shutil.copy("CMakeLists.txt",
                     os.path.join(self._source_subfolder, "CMakeLists.txt"))
 
+    def requirements(self):
+        self.requires("zlib/1.2.11")
+        self.requires("zstd/1.4.3")
```

Comment:
Why are this requirements needed now? Was this a bug?

Reply:
they are optional, but not listed in option because, eventually, the recipe will collect from the system. Also, now we have both libraries ported in CCI, probably they were not ready when libtiff was ported.